packages/dbadmin/src/controllers/DBAdminApi.ts

```diff
@@ -87,11 +87,12 @@ export class DBAdminApiImpl implements DBAdminApi {
         requestLogger.info(` * createRepository request received, with body of ${request.headers["content-length"]} bytes params: ${JSON.stringify(request.query)}`)
         const repositoryData: RepositoryData = { clientId: getClientIdParameter(request), repository: getRepositoryParameter(request) }
         const history = getHistoryParameter(request)
+        const lionWebVersion = getLionWebVersionParameter(request)
```

Comment:
Why isn't the LionWeb version part of `RepositoryData`?

Reply:
The reason for this is that the lionweb version is only needed when creating the repository.  After that, each store/delete should check its lionweb version against the one of the repository.